packages/relayer/http/get_events_by_address.go

```diff
@@ -0,0 +1,27 @@
+package http
+
+import (
+	"errors"
+	"math/big"
+	"net/http"
+
+	"github.com/cyberhorsey/webutils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/labstack/echo/v4"
+)
+
+func (srv *Server) GetEventsByAddress(c echo.Context) error {
+	chainID, ok := new(big.Int).SetString(c.QueryParam("chainID"), 10)
+	if !ok {
```

Comment:
I dont think we need to worry about things like Pagination for these endpoints. It is unlikely a single user will have so many (thousands) of results that we would need to worry about that at this point, seems liek overkill, so I left it simple.

Reply:
Agreed, but do we provide the option of "load more"?